File: examples/sendEmail.py
import requests
from urllib.parse import quote

payload = {
    'fromEmail' : 'myGmail@gmail.com',
    'to_email' : 'reciverGmail@gmail.com',
    'subject' : quote('About Nice Weather!'),
    'message' : quote('Its such a nice weather lets hang out'),
    'emailPassword' : 'xsydtdiyiudgiud'
}

response = requests.get('https://yourdomain.com', params=payload)
print(response.text)

File: examples/sendEmail.php
<?php
$your_email = 'myGmail@gmail.com';
$to_email = 'reciverGmail@gmail.com';
$subject = urlencode('About Nice Weather!');
$message = urlencode('Its such a nice weather lets hang out');
$emailPassword = 'xsydtdiyiudgiud';

$url = "http://domain.com/?fromEmail=$your_email&toEmail=$to_email&emailSubject=$subject&emailMessage=$message&emailPassword=$emailPassword";
$result = file_get_contents($url);
print_r($result);
?>
